pkg/cmd/server/start/admission.go

```diff
@@ -41,6 +41,7 @@ var (
 
 func init() {
 	defaultOffPlugins.Insert("AlwaysPullImages")
+	defaultOffPlugins.Insert("JenkinsBootstrapper")
```

Comment:
> I don't want to default this off. I want enabled to default to false.

Let's say we default it on (that allows it be set to off).  How would you distinguish between cases where ansible set the value to false and people were just waiting for it to be ready and the case where someone said "don't turn this on"?  That would eliminate the ability for us to enable the feature when its ready.

Defaulting to off now, does not preclude defaulting to on later.